src/backends/torch/torchinputconns.cc

```diff
@@ -965,17 +1065,34 @@ namespace dd
         }
   }
 
-  void CSVTSTorchInputFileConn::fill_dataset(TorchDataset &dataset,
-                                             bool use_csvtsdata_test)
+  void CSVTSTorchInputFileConn::fill_dataset()
```

Comment:
The id of passing TorchDataset to functions like this is to have a generic way of managing datasets, and making the code simpler.

I think this function could have been kept the way it was, and iteration on test set could be performed in ::transform.

Reply:
ok, refactored